src/memory/mmap_bytebuf.rs

```diff
@@ -5,8 +5,9 @@
 //! memory up to maximum. This might be a problem for systems that don't have a lot of virtual
 //! memory (i.e. 32-bit platforms).
 
-use std::ptr::{self, NonNull};
-use std::slice;
+use alloc::string::{String, ToString};
+use core::ptr::{self, NonNull};
+use core::slice;
```

Comment:
Strictly speaking we don't need this change because `virtual_memory` activates `std`.